apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -498,9 +555,12 @@ def render(element_html, data):
             except FileNotFoundError:
                 return "The reference solution is not provided for this question."
 
-        if grading_method == "unordered":
+        if grading_method is GradingMethodType.UNORDERED:
             ordering_message = "in any order"
-        elif grading_method == "dag" or grading_method == "ranking":
+        elif (
+            grading_method == GradingMethodType.DAG
```

Comment:
```suggestion
            grading_method is GradingMethodType.DAG
```
Some comment for line below.

Reply:
done